src/test-utils/consoleErrorMock.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable flowtype/require-valid-file-annotation, no-console */
```

Comment:
From my understanding, Using both is pointless. Here is the docs https://flow.org/en/docs/config/options/#suppresscomment-regex-a-classtoc-idtoc-suppress-comment-regex-hreftoc-suppress-comment-regexa